tests/app/main/test_suppliers.py

```diff
@@ -2282,3 +2282,63 @@ def test_existing_org_size_sets_current_selection(self, data_api_client, existin
             doc = html.fromstring(res.get_data(as_text=True))
             selected_value = doc.xpath('//input[@name="organisation_size" and @checked="checked"]/@value')
             assert selected_value == expected_selection, 'The organisation size has not pre-populated correctly.'
+
+
+@mock.patch("app.main.views.suppliers.data_api_client", autospec=True)
+class TestSupplierEditTradingStatus(BaseApplicationTest):
+    def test_edit_organisation_size_page_loads(self, data_api_client):
+        with self.app.test_client():
+            self.login()
+
+            res = self.client.get("/suppliers/trading-status/edit")
+            assert res.status_code == 200, 'The edit trading-status page has not loaded correctly.'
+
+    def test_no_selection_triggers_input_required_validation(self, data_api_client):
+        with self.app.test_client():
+            self.login()
+
+            res = self.client.post("/suppliers/trading-status/edit")
+            doc = html.fromstring(res.get_data(as_text=True))
+            error = doc.xpath('//span[@id="error-trading_status"]')
```

Comment:
Add assertion for validation masthead here :)

Reply:
Yep